model-optimizer/extensions/front/onnx/maxunpoolONNX_to_maxunpool.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.MaxUnpool import MaxUnpool
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.graph.graph import Graph
+
+class MaxUnpoolFrontReplacer(FrontReplacementSubgraph):
+    enabled = True
+
+    def pattern(self):
+        return dict(
+            nodes=[
+                ('max_pool0', dict(op='MaxPool')),
+                ('max_pool1', dict(op='MaxPool')),
+                ('slice', dict(op='AttributedSlice')),
+                ('sub', dict(op='Sub')),
+                ('unpool', dict(op='Unpooling')),
+            ],
+            edges=[
+                ('max_pool1', 'slice'),
+                ('max_pool0', 'sub', {'in': 0}),
+                ('slice', 'sub', {'in': 1}),
+                ('sub', 'unpool', {'in': 1}),
+            ])
```

Comment:
for the MO part:
we should extract onnx operator separately to some operation -- if it doesn't look like OV operation -- it should be private internal operation -- then this transformation takes place and creates Unpool. 
Also, we should add support of MaxUnpool to onnx frontend -- as it will become default engine for onnx model translation inside the MO.